### PR TITLE
Add webapp

### PR DESCRIPTION
**Pull request by atatarenko**

### CODE REVIEW

File: src/main/java/by/triumgroup/recourse/configuration/security/ResourceServerConfiguration.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public void configure(ResourceServerSecurityConfigurer resources) {
     public void configure(HttpSecurity http) throws Exception {
         http
                 .authorizeRequests()
-                .antMatchers("/user/**").hasAuthority("ORGANIZER")
-                .anyRequest().authenticated();
+                    .antMatchers("/**").permitAll()
```
Comment (ishymko) [inline src/main/java/by/triumgroup/recourse/configuration/security/ResourceServerConfiguration.java]: Should be configured later. Maybe add a TODO?

File: src/main/java/by/triumgroup/recourse/configuration/security/UserAuthDetails.java [hunk 2]
```diff
@@ -48,6 +48,6 @@ public boolean isCredentialsNonExpired() {
 
     @Override
     public boolean isEnabled() {
-        return !isDeleted();
+        return true;
```
Comment (ishymko) [inline src/main/java/by/triumgroup/recourse/configuration/security/UserAuthDetails.java:51]: Can we just remove method overriding?

File: src/main/webapp/app/propertyFilter.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+angular
+    .module('app')
+    .filter('propertyFilter', function() { return PropertyFilter});
+
+function PropertyFilter(items, props) {
```
Comment (ishymko) [inline src/main/webapp/app/propertyFilter.js:5]: I was not able to understand this function...

**Review by ishymko — CHANGES_REQUESTED** [review]

File: src/main/java/by/triumgroup/recourse/configuration/security/UserAuthDetails.java (hunk 2 above)
Comment (Meosit) [reply src/main/java/by/triumgroup/recourse/configuration/security/UserAuthDetails.java:51]: @vanashimko Nope, it required by Spring

Comment (atatarenko) [reply src/main/java/by/triumgroup/recourse/configuration/security/UserAuthDetails.java:51]: nop

Comment (ishymko) [reply src/main/java/by/triumgroup/recourse/configuration/security/UserAuthDetails.java:51]: @Meosit I'm very glad to see you

**Review by ishymko — APPROVED** [review]

File: src/main/java/by/triumgroup/recourse/service/impl/AbstractCrudService.java [hunk 4]
```diff
@@ -44,4 +44,9 @@ protected AbstractCrudService(CrudRepository<E, ID> repository) {
         }
         return result;
     }
+
+    @Override
+    public Iterable<E> findAll() {
+        return wrapJPACall(() -> repository.findAll());
```
Comment (Meosit) [inline src/main/java/by/triumgroup/recourse/service/impl/AbstractCrudService.java:50]: Could be replaced with `wrapJPACall(repository::findAll);`

File: src/main/webapp/app/propertyFilter.js (hunk 3 above)
Comment (atatarenko) [reply src/main/webapp/app/propertyFilter.js:5]: It's just magic function from stackoveflow. We need this for autocomplete.
When you are typing in select box text field, we should hide such items, which don't suit us, just filter them.
By default we can filter items this way `item | filter { name: $select.search, age: $select.search}`. It works like **and** for name and age. Using default filter we say to ui-select: hey, just show me all items where age fits entered string and **at the same time** age fits entered string.
But we need **or** here. So with this strange function we can ask again: hey, show me such items where name fits entered string **or** age fits one. 

P.S. Nice comment. Finally realized why this function is needed :)

Comment (ishymko) [reply src/main/webapp/app/propertyFilter.js:5]: Age? Why age?

Comment (ishymko) [reply src/main/webapp/app/propertyFilter.js:5]: Oh, it was just an example. Thanks, it's clear now

File: src/main/java/by/triumgroup/recourse/service/impl/AbstractCrudService.java (hunk 4 above)
Comment (atatarenko) [reply src/main/java/by/triumgroup/recourse/service/impl/AbstractCrudService.java:50]: couldn't. `repository.findAll()` is overloaded and can accept `WrapperVoidFunction` and `WrapperFunction<Iterable<E>>`, so need casting here. `wrapJPACall((WrapperFunctions.WrapperFunction<Iterable<E>>) repository::findAll)` looks ugly, isn't it?

Comment (Meosit) [reply src/main/java/by/triumgroup/recourse/service/impl/AbstractCrudService.java:50]: Agree with you, didn't notice it

**Review by Meosit — APPROVED** [review]

**Pull request merged**